### ISSUE TITLE
Make it possible to format workspace names

### ISSUE THREAD

**Issue by Ventajou** [folke/vscode-monorepo-workspace#85]
What I'm looking for is simply to exclude the org name that's used in the package.json name from the displayed label.

For example in our repo all of our package names are prefixed `@foo/` but it becomes very redundant when that's used as workspace name.

Maybe some combination of regexp/template would allow users to configure package names.

**Comment by ujwal-setlur** [comment]
that would be useful!

**Comment by akrawitz** [comment]
Even just a checkbox to exclude the scope from the name would be great!

**Comment by tom-fletcher** [comment]
Agreed that removing the scope would be great, as it is redundant in a repo where all the scope is the same for every package.

@folke I have submitted a PR that adds an option for this
